packages/protocol-kit/src/contracts/safeDeploymentContracts.ts

```diff
@@ -175,3 +176,25 @@ export async function getSimulateTxAccessorContract({
   }
   return simulateTxAccessorContract
 }
+
+export async function getSafeWebAuthnSignerFactoryContract({
+  safeProvider,
+  safeVersion,
+  customContracts
+}: GetContractInstanceProps): Promise<SafeWebAuthnSignerFactoryContractImplementationType> {
+  const safeWebAuthnSignerFactoryContract = await safeProvider.getSafeWebAuthnSignerFactoryContract(
+    {
+      safeVersion,
+      customContractAddress: customContracts?.simulateTxAccessorAddress,
```

Comment:
Should be like this, right?
```suggestion
      customContractAddress: customContracts?.safeWebAuthnSignerFactoryAddress,
      customContractAbi: customContracts?.safeWebAuthnSignerFactoryAbi
```

Reply:
done!